openfga/v2beta1/openfga_service.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+package openfga.v2beta1;
+
+import "google/protobuf/struct.proto";
+import "openfga/v2beta1/openfga.proto";
+
+service OpenFGAService {
+    rpc Check(CheckRequest) returns (CheckResponse);
+    rpc ListObjects(ListObjectsRequest) returns (ListObjectsResponse);
+
+    rpc DeleteRelationships(DeleteRelationshipsRequest) returns (DeleteRelationshipsResponse);
+    rpc WriteRelationships(WriteRelationshipsRequest) returns (WriteRelationshipsResponse);
+
+    rpc WriteModel(WriteModelRequest) returns (WriteModelResponse);
+    rpc ReadModel(ReadModelRequest) returns (ReadModelResponse);
+    rpc ListModels(ListModelsRequest) returns (ListModelsResponse);
```

Comment:
ReadChanges?
Read?
BatchCheck?

Reply:
More to come 😄 . Just started with some.